source/1_engine/rex_engine/src/diagnostics/logging/internal/details/formatting/flag_formatter.cpp

```diff
@@ -0,0 +1,214 @@
+#include "rex_engine/diagnostics/logging/internal/details/formatting/flag_formatter.h"
+#include "rex_engine/diagnostics/logging/internal/details/formatting/padder.h"
+#include "rex_engine/diagnostics/logging/internal/details/fmt_helper.h"
+
+namespace rexlog
+{
+    namespace details
+    {
+        ///////////////////////////////////////////////////////////////////////
+        // Time information
+        ///////////////////////////////////////////////////////////////////////
+        const rsl::array<const char*, 7>    DayNames::days = { {"Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"} };
```

Comment:
I think it's best to use string_view as the array value type here and for the other arrays as well